doc/source/programs/gdal_sozip.rst

```diff
@@ -0,0 +1,267 @@
+SOZip-compressed files. Defaults to ``ALL_CPUS``.
+
+Examples
+++++++++
+
+.. example::
+   :title: Create a, potentially seek-optimized, ZIP file with the content of my.gpkg
+
+   .. code-block:: bash
+
+        gdal sozip create my.gpkg my.gpkg.zip
+
+.. example::
+   :title: Create a, potentially seek-optimized, ZIP file from the content of a source directory:
+
+   .. code-block:: bash
+
+       gdal sozip create -rsource_dir/ my.gpkg.zip
+
+
+"gdal sozip optimize"
+---------------------
+
+Description
++++++++++++
+
+Create a new zip file from the content of an existing one, possibly applying
+SOZip optimization when relevant.
+
+Synopsis
+++++++++
+
+.. program-output:: gdal sozip optimize --help-doc
+
+Options
++++++++
+
+.. option:: -i, --input <INPUT>
+
+    Input ZIP filename. Required.
+
+.. option:: -o, --output <OUTPUT>
+
+    Output ZIP filename. Required. Must have a ``.zip`` extension
+
+.. option:: --overwrite
+
+    Whether overwriting existing output is allowed.
+
+.. option:: --enable-sozip auto|yes|no
+
+    In ``auto`` mode, a file is seek-optimized only if its size is above the
+    value of :option:`--sozip-chunk-size`.
+    In ``yes`` mode, all input files will be seek-optimized.
+    In ``no`` mode, no input files will be seek-optimized.
+
+.. option:: --sozip-chunk-size <value>
+
+    Chunk size for a seek-optimized file. Defaults to 32768 bytes. The value
+    is specified in bytes, or K and M suffix can be respectively used to
+    specify a value in kilo-bytes or mega-bytes.
+
+.. option:: --sozip-min-file-size <value>
+
+    Minimum file size to decide if a file should be seek-optimized, in
+    --enable-sozip=auto mode. Defaults to 1 MB byte. The value
+    is specified in bytes, or K, M or G suffix can be respectively used to
+    specify a value in kilo-bytes, mega-bytes or giga-bytes.
+
+.. option:: -q, --quiet
+
+    Do not output any informative message (only errors).
+
+Multithreading
+++++++++++++++
+
+The :config:`GDAL_NUM_THREADS` configuration option can be set to
+``ALL_CPUS`` or a integer value to specify the number of threads to use for
+SOZip-compressed files. Defaults to ``ALL_CPUS``.
+
+Examples
+++++++++
+
+.. example::
+   :title: Create a, potentially seek-optimized, ZIP file ``sozip_optimized.zip`` from an existing ZIP file ``in.zip``.
+
+   .. code-block:: bash
+
+        gdal sozip optimize in.zip sozip_optimized.zip
+
+
+"gdal sozip list"
+-----------------
+
+Description
++++++++++++
+
+List the files contained in the zip file in an output similar to Info-ZIP
+:program:`unzip` utility, but with the addition of a column indicating
+whether each file is seek-optimized.
+
+Synopsis
+++++++++
+
+.. program-output:: gdal sozip list --help-doc
+
+Options
++++++++
+
+.. option:: -i, --input <INPUT>
+
+    Input ZIP filename. Required.
+
+Examples
+++++++++
+
+.. example::
+   :title: List contents of ``my.zip``.
+
+   .. code-block:: bash
+
+        gdal sozip list my.zip
```

Comment:
A few lines above `-i` or `--input` was required

Reply:
This is a positional argument. Positional arguments in the argument framework can be either specified as optional arguments or directly. So "gdal sozip list my.zip", "gdal sozip list --input my.zip" or "gdal sozip list -i my.zip" are all possible